packages/cli/src/lib/setup/setupInstall.ts

```diff
@@ -750,19 +797,22 @@ function Install(options) {
         const instanceObj = deepClone(doc);
 
         instanceObj._id = `system.adapter.${adapter}.${instance}`;
+        // @ts-expect-error we now convert the adapter object to an instance object
         instanceObj.type = 'instance';
+        // @ts-expect-error types needed TODO
         if (instanceObj.common.news) {
+            // @ts-expect-error types needed TODO
             delete instanceObj.common.news; // remove this information as it could be big, but it will be taken from repo
         }
-        if (instanceObj._rev) {
-            delete instanceObj._rev;
-        }
+
         instanceObj.common.enabled =
             options.enabled === true || options.enabled === false
                 ? options.enabled
                 : instanceObj.common.enabled === true || instanceObj.common.enabled === false
                 ? instanceObj.common.enabled
                 : false;
```

Comment:
```suggestion
        instanceObj.common.enabled = options.enabled ?? instanceObj.common.enabled ?? false;
```

Reply:
On 2nd thought, this does not repair wrong types, e.g. `"true"`, so maybe we don't want this.